content/en/docs/setup/additional-setup/external-controlplane/index.md

```diff
@@ -100,10 +96,19 @@ spec:
 EOF
 {{< /text >}}
 
-Apply the configuration in the `istio-system` namespace of `external_cluster`:
+Install the configuration to create the ingress gateway in the `istio-system` namespace of `external_cluster`:
 
 {{< text bash >}}
-$ istioctl apply -f controlplane-gateway.yaml --context="${CTX_EXTERNAL_CLUSTER}"
+$ istioctl install -f controlplane-gateway.yaml --context="${CTX_EXTERNAL_CLUSTER}"
+{{< /text >}}
+
+You may notice the istiod deployment created in the istio-system namespace. This is used only to configure the ingress gateway and is NOT the control plane used by remote clusters.
```

Comment:
Maybe add another sentence about the actual external istiod, something like this?
```suggestion
You may notice the istiod deployment created in the istio-system namespace. This is used only to configure the ingress gateway and is NOT the control plane used by remote clusters.
This ingress gateway could, in fact, be configured to host multiple external control control planes, in different namespaces on the cluster, even though in this example you will only deploy a single external istiod in the `external-istiod` namespace.
```